src/application/machines/reconnectMachine.vscode.tsx

```diff
@@ -0,0 +1,18 @@
+import { setup } from "xstate";
+export const reconnectMachine = setup({
+  delays: { connectTimeout: 500 },
+}).createMachine({
```

Comment:
In VSCode, we want to show this from the start, as it has valuable information.
We will wait for a very short amount of time though, in case something immediately connects - that way we avoid flickering of the modal.